app.js

```diff
@@ -1 +1,35 @@
-console.log("Todo List NodeJS!")
\ No newline at end of file
+
+
+const readline = require('readline'); //console.info(readline);
+let userQuestions = readline.createInterface(process.stdin,process.stdout);
+let decided = readline.createInterface(process.stdin, process.stdout);
+
+function getUserWant(){
+    "use strict";
+    const activities = [{
+        monday : "playing soccer",
+        tuesday : "playing volley",
+        wednesday : "playing basketball",
+        thursday : "playing shooter game",
+        friday : "walk around with pets",
+        saturday : "hanging out with my Boo or with homies",
+        sunday : "sleeping over all times"
+    }];
```

Comment:
What's this? I don't think is even related to the task at hand. Please delete this.